CHANGELOG.md

```diff
@@ -13,10 +13,11 @@ All notable changes to this project will be documented in this file.
 * Upgrade App Engine Plugins Core dependency to 0.5.2.
 * `cloudSdkPath` has been replaced with `cloudSdkHome`. ([#257](../../issues/257))
 * Remove deprecated `appYamls` parameter. ([#162](../../issues/162))
-* `project` and `version` are no longer pulled from the global gcloud state. `project` must be configured
-in either pom.xml using the `<project>` tag or in appengine-web.xml using the `<application>` tag. 
-`version` may be left unconfigured to be generated by gcloud, or it may be configured in your pom.xml or 
-appengine-web.xml. To read values from appengine-web.xml, set the system property `deploy.read.appengine.web.xml=true`. ([#285](../../issues/285))
+* `project` and `version` are no longer pulled from the global gcloud state by default. `project`
```

Comment:
I think this might need to made more maven specific

`<project>APPENGINE_CONIFG</project>` vs `project = "APPENGINE_CONFIG"`